src/inet/UDPEndPoint.cpp

```diff
@@ -136,7 +136,8 @@ namespace {
 
 CHIP_ERROR CheckMulticastGroupArgs(InterfaceId aInterfaceId, const IPAddress & aAddress)
 {
-    VerifyOrReturnError(aInterfaceId.IsPresent(), INET_ERROR_UNKNOWN_INTERFACE);
+    // Skip verification of Interface since IsPresent() always fail
+    // VerifyOrReturnError(aInterfaceId.IsPresent(), INET_ERROR_UNKNOWN_INTERFACE);
```

Comment:
Either we remove it, or we open an explicit issue to fix this.

@kpschoedel Should this be removed?

Reply:
Probably; we clearly want to use the code without one, and I didn't find any explanation for the check in the ancestral code.